vocab/properties.json

```diff
@@ -2,8 +2,9 @@
     "https://openminds.ebrains.eu/vocab/IRI": {
         "description": "Stands for Internationalized Resource Identifier which is an internet protocol standard that builds on the URI protocol, extending the set of permitted characters to include Unicode/ISO 10646.",
         "label": "IRI",
-        "labelForReverseLink": "Is IRI of",
+        "labelForReverseLink": "Identifies",
```

Comment:
We avoided so far using verbs in properties and I think we should stick to that (we can re-discuss though). A generic solution would be "isIdentifierOf".

Reply:
I think reverse links should not be more generic than the actual link (this could lead to conflicts). I would stick with "isIRIOf"